user/installing-dependencies.md

```diff
@@ -303,6 +303,26 @@ rvm use $TRAVIS_RUBY_VERSION # optionally, switch back to the Ruby version you n
 
 > You can also have a look at the [Homebrew](https://config.travis-ci.com/ref/job/addons/homebrew) section in our [Travis CI Build Config Reference](https://config.travis-ci.com/).
 
+## Installing Packages on FreeBSD
+
+To install packages that are not included in the default FreeBSD environment use `pkg` in the `before_install` step of your `.travis.yml`:
+
+```yaml
+before_install:
+  - sudo pkg install -y curl
```

Comment:
```suggestion
  - su -m root -c 'pkg install -y curl'
```
FreeBSD [doesn't ship](https://docs.freebsd.org/en/books/handbook/security/#security-sudo) with the `sudo` utility. It wasn't present in the Travis image when I checked back in January 2021; not sure if Travis includes it now.